plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizard.java

```diff
@@ -305,4 +325,44 @@ private static void addNature(IProject project) throws CoreException {
     public void init(IWorkbench workbench, IStructuredSelection selection) {
         this.selection = selection;
     }
+
+    private String changeXMLName(String content, String taskName) throws CoreException {
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+        Document dom = null;
+        String xmlString = null;
+        try {
+
+            DocumentBuilder db = dbf.newDocumentBuilder();
+            InputSource is = new InputSource(new StringReader(content));
+            dom = db.parse(is);
+            NodeList taskList = dom.getElementsByTagName(HumantaskEditorConstants.QUALIFIED_TASK_NODE_NAME);
+            for (int i = 0; i < taskList.getLength(); i++) {
+                Node task = taskList.item(i);
+                task.getAttributes().getNamedItem("name").setNodeValue(taskName);
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/3b81f244b662cbce6b9e7ed2d7c1015f2edf20d0